### PR TITLE
Add sfp_company module to Passive category

### PR DESCRIPTION
**Pull request by bcoles**

### CODE REVIEW

File: modules/sfp_company.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 from sflib import SpiderFoot, SpiderFootPlugin, SpiderFootEvent
 
 class sfp_company(SpiderFootPlugin):
-    """Company Names:Footprint,Investigate,Passive:Content Analysis::Identify company names in any obtained data."""
+    """Company Names:Passive,Footprint,Investigate,Passive:Content Analysis::Identify company names in any obtained data."""
```
Comment (smicallef) [inline modules/sfp_company.py]: Passive is already there :)

Comment (bcoles) [reply modules/sfp_company.py]: Welp. You're right. I didn't even notice. It looks like there's a bug lurking here.

Despite `Found company name` appearing in the logs, and other modules declaring receipt of a `COMPANY_NAME` event, the company name does not appear in the `Browse` tab, nor when exporting the results as CSV.

Comment (smicallef) [reply modules/sfp_company.py]: Make sure you're not using an old version of the database..

Comment (bcoles) [reply modules/sfp_company.py]: Fair call. That fixed it. Thanks.

**Pull request closed without merging**